python/lsst/ts/externalscripts/maintel/make_lsstcam_calibrations.py

```diff
@@ -198,3 +234,45 @@ def get_schema(cls):
 
     def get_instrument_configuration(self):
         return dict(filter=self.config.filter)
+
+    async def assert_feasibility(self, image_type):
+        """Ensure dome components are in the required state before flats."""
+        if image_type != "FLAT":
+            return None
+
+        # Check if dome components are ignored by user configuration.
+        if (
+            hasattr(self._mtcs.check, "mtdometrajectory")
+            and not self._mtcs.check.mtdometrajectory
+        ):
+            raise RuntimeError(
```

Comment:
We should allow people to ignore mtdometrajectory, please remove this exception.